codec-http/src/main/java/io/netty5/handler/codec/http/headers/HttpHeaderValidationUtil.java

```diff
@@ -0,0 +1,302 @@
+/*
+ * Copyright 2022 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty5.handler.codec.http.headers;
+
+import io.netty5.handler.codec.http.HttpHeaderNames;
+import io.netty5.handler.codec.http.HttpHeaderValues;
+import io.netty5.util.AsciiString;
+import io.netty5.util.internal.UnstableApi;
+
+import static io.netty5.util.AsciiString.contentEqualsIgnoreCase;
+
+/**
+ * Functions used to perform various validations of HTTP header names and values.
+ */
+@UnstableApi
```

Comment:
Imho we should have `@UnstableApi` in netty 5. Either its "stable" or if its internal use `@Internal`
`